package.json

```diff
@@ -71,7 +71,7 @@
     "move-concurrently": "^1.0.1",
     "promise-inflight": "^1.0.1",
     "rimraf": "^2.6.3",
-    "ssri": "^6.0.1",
+    "ssri": "^8.0.1",
```

Comment:
v6.0.2 is released, we should be able to use that here.

https://github.com/npm/ssri/issues/18#issuecomment-815221327

Reply:
Awesome, thanks! I'll update and hopefully that'll break fewer tests :)

Reply:
That did the trick!